src/main.rs

```diff
@@ -270,6 +270,21 @@ enum Opt {
         #[structopt(short, long, parse(from_os_str))]
         out: Option<PathBuf>,
     },
+    /// Create a new cargo project in an existing directory
+    #[structopt(name = "init")]
+    InitProject {
+        /// Project path
+        path: Option<String>,
+        /// Set the resulting package name, defaults to the directory name
+        #[structopt(long)]
+        name: Option<String>,
+        /// Use mixed Rust/Python project layout
+        #[structopt(long)]
+        mixed: bool,
+        /// Which kind of bindings to use
+        #[structopt(short, long, possible_values = &["pyo3", "rust-cpython", "cffi", "bin"])]
+        bindings: Option<String>,
```

Comment:
I think we can extract `name`, `mixed` and `bindings` to a seperated struct and use `#[structopt(flatten)]` to reuse them. For example 
https://github.com/PyO3/maturin/blob/fc039bd671430c180589b7d9d8ed7d548cf94798/src/main.rs#L197-L199

Reply:
👍 will do this and fix formatting tonight